comp_loinc/build.py

```diff
@@ -1,83 +1,157 @@
+    lcc.write_output_to_file("./data/output/owl_component_files/code_classes.owl")
     """
-    ## Example
-    # lcc = CodeIngest("./model/schema/code_schema.yaml", "./data/part_files")
-    # lcc.write_output_to_file("./data/output/owl_files/code_classes.owl")
     lcc = CodeIngest(schema_file, part_directory)
     lcc.write_output_to_file(output)
 
 
 @app.command(name='composed')
 def build_composed_classes(
-        schema_file: str,
-        composed_classes_data_file: str,
-        output: str):
-    """
-    todo: this is just calling a linkml owl cli, should be written as code with python
-    :param schema_file:
-    :param code_file:
-    :param output:
-    :return:
+    schema_file: Path = typer.Option(default=DEFAULTS['schema_file.composed'], resolve_path=True, exists=False),
+    composed_classes_data_file: Path = typer.Option(
+        default=DEFAULTS['composed_classes_data_file'], resolve_path=True, exists=False),
+    output: Path = typer.Option(default=DEFAULTS['output.composed'], resolve_path=True, writable=True)
+):
+    """Build composed classes ontology.  Part 3/5 of the pipeline.
+
+    :param schema_file: Path to LinkML `.yaml` file that defines data model for "grouping classes" of LOINC terms, that 
+    is, classes that group sets of LOINC terms into specific categories.
+    :param composed_classes_data_file: Path to `.yaml` file which lists LOINC composed classes. These are lower-level,
+    more granular groupings of classes, and their are a greater number of them than the grouping classes in the
+    `schema_file`.
+    :param output: Path where output will be saved.
+
+    todo: this is just calling a linkml owl cli, should be written as code with python code
     """
     subprocess.call(["linkml-data2owl", "-s", schema_file, composed_classes_data_file, "-o", output])
 
+
 @app.command(name="merge")
 def merge_owl(
-        owl_directory: str,
-        output: str):
-    # TODO: Consider removing the files created from this point on each time this code executes e.g. any file with 'merge_*'
-    files = [f'{owl_directory}{x}' for x in os.listdir(owl_directory) if ".owl" in x]
+    owl_directory: Path = typer.Option(default=DEFAULTS['owl_directory'], resolve_path=True, exists=False),
+    output: Path = typer.Option(default=DEFAULTS['output.merge'], resolve_path=True, writable=True)
+):
+    """Merge all OWL ontology files into a single ontology. Part 4/5 of the pipeline.
+
+    :param owl_directory: Path to directory where unmerged `.owl` files are stored.
+    :param output: Path where output will be saved.
+
+    TODO: Consider removing the files created from this point each time this code executes e.g. any file with 'merge_*'
+    """
+    files = [f'{owl_directory}{x}' for x in os.listdir(owl_directory) if ".owl" in str(x)]
     subprocess.call(["./robot", "merge", "-i"] + " -i ".join(files).split() + ['-o', output])
 
+
 @app.command(name="reason")
 def reason_owl(
-        owl_directory: str,
```

Comment:
From `reason_owl()`: removed `owl_directory`, as the directory where merged and umerged OWL files was different but ambiguous.

@putmantime Based on looking at the files and seeing some duplicates, I take it that you had not yet finalized this directory structure. For now, I've renamed `owl_files` to `owl_component_files` and changed it so that unmerged files are in `output/owl_component_files/` and merged OWL files go directly into `output/`